fc/parsing/actions.py

```diff
@@ -1002,6 +1060,21 @@ def _convert_output_units(self):
             output_variables.update(variables)
         return output_variables
 
+    def _list_parameters(self):
+        """Populates the list of model parameters (inputs with a constant RHS)."""
+
+        for var in self.inputs:
+            try:
+                variable = self.model.get_variable_by_ontology_term(var.rdf_term)
+            except KeyError:
+                # Skip optional variables
+                continue
+
+            # Parameters are inputs that aren't states, and have a constant RHS
+            eq = self.model.get_definition(variable)
+            if isinstance(eq.lhs, VariableDummy) and len(eq.rhs.atoms(VariableDummy)) == 0:
```

Comment:
Not something to do in this PR, but perhaps worth discussing with @MauriceHendrix whether a `Model.is_parameter(self, variable)` method that does this in cellmlmanip would be useful?